en/flight_modes_rover/manual.md

```diff
@@ -0,0 +1,88 @@
+# Manual Modes (Rover)
+
+Manual modes require stick inputs from the user to drive the vehicle.
+
+![Manual Controls](../../assets/airframes/rover/flight_modes/rover_manual_controls.png)
+
+The manual modes listed below provide increasing levels of autopilot support:
+
+| Mode                           | Features                                                                                                   |
+| ------------------------------ | ---------------------------------------------------------------------------------------------------------- |
+| [Manual](#manual-mode)         | Directly map stick inputs to motor commands, no closed loop control.                                       |
+| [Acro](#acro-mode)             | Closed loop yaw rate control.                                                                              |
```

Comment:
1. Three column table showing what is supported for Differential, Ackermann, Features? (like your other table)
2. What are open loop and closed loop yaw rate control? (Hint, I do know, but the readers may not, and for most of this _should not need to_). 

   For this whole doc, the audience here is as far as possible a high school kid. Assume they know as little as possible about control theory as possible. Minimally it should _start_ with this assumption, and only go to the detail when you absolutely have to. You can look at the FC and MC docs for inspiration. 
   
   You want to give a flavour to what this means for driving - i.e. how does closed loop yaw rate control make acro "better" than manual.
   
   This is like the dummies version I mention below https://github.com/PX4/PX4-user_guide/pull/3377/files#r1764257980

   "Yaw rate is controlled, providing better cornering ..."

Hope that makes sense?

Reply:
Updated this table with more and easier to understand information [here](https://github.com/PX4/PX4-user_guide/blob/cd3233a9ace5fe1a14e95b1a4081b57958c60c55/en/flight_modes_rover/differential.md?plain=1#L21-L26).